integration_tests/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "noEmit": true,
```

Comment:
we actually don't need `tsc` generate any JS since Cypress will do that for us when it transpires the source